src/types/types.ts

```diff
@@ -824,10 +853,17 @@ export interface FileFilter extends Limit {
      * Only include files that reference these specific asset IDs.
      */
     assetIds?: CogniteInternalId[];
+    /**
+     * Only include files that have a related asset in a tree rooted at any of these root assetIds.
```

Comment:
This comment is not consistent across types? In one place I see:

"The IDs of the root assets that the related assets should be children of." -- if the behaviour is the same they should probably have consistent comments.

Reply:
agree